src/main/java/com/glovo/ttglovo/cart/CartItem.java

```diff
@@ -0,0 +1,59 @@
+package com.glovo.ttglovo.cart;
+
+import com.glovo.ttglovo.securityManagement.appuser.AppUser;
+import lombok.*;
+
+import javax.persistence.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity(name = "CartItem")
+@Table(name = "cart_items")
+public class CartItem {
+
+    @Id
+    @SequenceGenerator(
+            name = "cart_sequence",
+            sequenceName = "cart_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "cart_sequence"
+    )
+    @Column(name = "id", updatable = false, nullable = false)
+    private Long id;
+
+    @Column(name = "meal_id")
+    private Long mealId;
+
+    @Column(
+            name = "meal_price",
+            nullable = false,
+            columnDefinition = "Integer")
```

Comment:
If not more than ~100 characters long, it's better to keep it in one line. In the model class this keeps the column definitions more tight.